mettle/api/api.go

```diff
@@ -72,17 +69,24 @@ func serve(opts grpcutil.Opts, name, requestQueue, responseQueue, preResponseQue
 		preResponses: common.MustOpenTopic(preResponseQueue),
 		jobs:         map[string]*job{},
 	}
+
 	go srv.Receive()
+	if jobs, err := getExecutions(opts, true); err != nil {
```

Comment:
why is conTLS always true? We would not want this internally for example.

Reply:
Good point, this would usually be false right? I've added a TLS flag to the api options and set default: false.